homework-submission/blog/src/Comments.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Update from './Update';
+
+class Comments extends React.Component {
+  state = {
+    update: false,
+    number: 0,
+  };
+  handleEdit = () => {
+    this.setState({ update: true });
+  };
+  handleCancel = () => {
+    this.setState({ update: false });
+  };
+  handleUpdate = (updatedItem, id) => {
+    this.props.handleUpdate(updatedItem, id);
+    this.setState({ update: false });
+  };
+  handleClick = () => {
+    this.setState({ number: this.state.number + 1 });
+    this.props.item.isLiked = true;
+  };
```

Comment:
and this function you are increasing the likes counter, but it should swap between like and unlike